pyclient/pydeephaven/session.py

```diff
@@ -67,6 +68,7 @@ def __init__(self, host: str = None, port: int = None, never_timeout: bool = Tru
         self._tables = {}
         self._never_timeout = never_timeout
         self._keep_alive_timer = None
+        self.session_type = session_type
```

Comment:
Should we try to stick to an underscore for internal variables coding convention?

Reply:
Yeah that's fine, updated